torch/csrc/dynamo/guards.cpp

```diff
@@ -54,8 +54,8 @@ class TensorCheck {
     if (ndim != dim_) {
       return false;
     }
-    const auto& sizes = v.sizes();
-    const auto& strides = v.strides();
+    const auto& sizes = v.sym_sizes();
+    const auto& strides = v.sym_strides();
```

Comment:
🤔  This is the right thing to do but it may make guard perf worse. Hopefully not too much worse.